Common/MathUtils/src/Cartesian3D.cxx

```diff
@@ -0,0 +1,44 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See https://alice-o2.web.cern.ch/ for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "MathUtils/Cartesian3D.h"
+
+using namespace o2::Base;
+
+ClassImp(o2::Base::Transform3D)
```

Comment:
just a tiny comment here: It is better to put `;` behind ClassImp as otherwise clang-format does not work correctly **OR** to put ClassImp at the end of the file.